P6 cl156 small goalbuddy card styling

JeffreyChuCPA opened the pull request:
Implemented styling as per HiFi design 

✅Added RoleBadge component as per work done in the PR for #114 to ensure the same component will be used for both PRs
![image](https://github.com/user-attachments/assets/52f40aea-42f3-40f2-a1a8-61a33deed8f3)
![image](https://github.com/user-attachments/assets/7aadf85b-788a-4001-a9bc-f58a0178f457)

JeffreyChuCPA commented:
> comparing to the success screenshot of design's prototype ![image](https://private-user-images.githubusercontent.com/98787046/413998377-0cafd2a2-9b3a-4163-827e-80c357b1506b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4MjkxNDUsIm5iZiI6MTczOTgyODg0NSwicGF0aCI6Ii85ODc4NzA0Ni80MTM5OTgzNzctMGNhZmQyYTItOWIzYS00MTYzLTgyN2UtODBjMzU3YjE1MDZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjE3VDIxNDcyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4NjE3YjgzN2ZhODkyZmQ1MGE5ZDMzMDZmOGYxM2UwMzFlOTg2Mjk5Y2IxNDI0ZjA1ZWFhM2E5MjY0NWJlNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.F8NfU-8Y8xG325h7m_J_-rv4JREkcomPNFJXOLQV9mk)
> 
> -should not be a grey background to the skills? -colour blocks should not leave gaps but fill to the right on the top of a user's colab card -all avatar pics are grey circles. is this to be addressed in another ticket? or did the db reset erase all user pics? -font is different from prototype. is the specific data not available from design yet?
> 
> foreseeing future conflicts, both this PR and PR37 deal with the colour blocks. Rather than having two implementations of the same component, please be sure to confer with that ticket's owner (Kevin) in regards to the colour blocks to compare implementations and decide which should be merged.

- Pending design to see if that greybackground should be kept or not, it was my own visual take lol
- Pending what design says, i recall we discussed it to be like this but will confirm
- DB issue, I render the pic from whatever is saved in the user doc and some of them reference a link that doesnt work
- Specific data is available from figma, was thinking fonts would be dealt with in another ticket to make the variables for them and apply to all components

AD9-1 commented:
The color of the Networking is orange, not yellow. 
<img width="206" alt="image" src="https://github.com/user-attachments/assets/19727302-4892-486d-af9f-647f4a11b4a8" />
For the above card, if the blue n green dots are closer to each other, then it would be great.It does not look good.

JeffreyChuCPA commented:
> The color of the Networking is orange, not yellow. <img alt="image" width="206" src="https://private-user-images.githubusercontent.com/84341093/414037483-19727302-4892-486d-af9f-647f4a11b4a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4MzgzNzgsIm5iZiI6MTczOTgzODA3OCwicGF0aCI6Ii84NDM0MTA5My80MTQwMzc0ODMtMTk3MjczMDItNDg5Mi00ODZkLWFmOWYtNjQ3ZjRhMTFiNGE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjE4VDAwMjExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmMzYyYWRhZWUzNDZkMGI3M2M4YTEwZWM5OWZmNzIyOTE1ZDg4N2FmYTY3ZmQ3MDM4NDFlYTRhN2QwNTgzNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SWeQrICShjmFOuljJ2cUdNZkAe0CHX7RdxZ7TdgEfO8"> For the above card, if the blue n green dots are closer to each other, then it would be great.It does not look good.

Updated for the spacing as discussed in call, and updated the colors to match exactly to what was used in hi-fi: 
![image](https://github.com/user-attachments/assets/e9829fcc-48f1-4c69-ba9a-aa1f1d274dfc)

KevinMeisenbacher commented:
Looks good

Redi-Molla commented:
Aparna's profile started throwing errors after the issue with the database earlier. Just like Katrina said, you have to do git pull origin develop to see the nav bar at the top.

JeffreyChuCPA commented:
> Aparna's profile started throwing errors after the issue with the database earlier. Just like Katrina said, you have to do git pull origin develop to see the nav bar at the top.

Updated and merged with origin, no issues noted.

JeffreyChuCPA commented:
> did you ask design about the grey backgrounds?

Design said they were fine with it
